doc/source/user_guide/io.rst

```diff
@@ -611,6 +615,10 @@ If the header is in a row other than the first, pass the row number to
 Duplicate names parsing
 '''''''''''''''''''''''
 
+  .. deprecated:: 1.5.0
+     ``mangle_dupe_cols`` was never implemented, and a new argument where the
+     renaming pattern can be specified will be added instead.
+
```

Comment:
I'd personally update the section to just show that we rename columns, and remove all the references to `mangle_dupe_cols`, which never worked anyway.

Reply:
Sounds good to me. Removed the example with False